sae_lens/analysis/neuronpedia_integration.py

```diff
@@ -70,30 +70,71 @@ def open_neuronpedia_feature_dashboard(sae: SAE, index: int):
         webbrowser.open(url)
 
 
+class FeatureInfo(NamedTuple):
+    feature_index: int
+    description: str
+    model_name: Optional[str]
+    neuronpedia_id: Optional[str]
```

Comment:
these should be default `None`, otherwise users will have to provide `model_name = None` explicitly

Reply:
Thanks, fixed!